qa/rpc-tests/wallet.py

```diff
@@ -249,6 +249,19 @@ def run_test (self):
 
         assert_equal("not an integer" in errorString, True)
 
+        # Mine a block from node0 to an address from node1
+        cbAddr = self.nodes[1].getnewaddress()
+        blkHash = self.nodes[0].generatetoaddress(1, cbAddr)[0]
+        cbTxId = self.nodes[0].getblock(blkHash)['tx'][0]
+        self.sync_all()
+
+        # Check that the txid and balance is found by node1
+        try:
+            self.nodes[1].gettransaction(cbTxId)
+        except JSONRPCException,e:
+            errorString = e.error['message']
+        assert_equal("Invalid or non-wallet transaction id" in errorString, False)
```

Comment:
Uh, I don't think so. There is an error from gettransaction and the error string was defined earlier.

Anyways, this should work, right?
```
except JSONRPCException,e:
            assert("Invalid or non-wallet transaction id" not in e.error['message'])
```